website/docs/guides/sl-snowflake-qs.md

```diff
@@ -0,0 +1,870 @@
+
+### What you'll learn
+
+This guide will cover the following topics:
+
+- Create a new Snowflake worksheet and set up your environment
+- Load sample data into your Snowflake account
+- Connect dbt Cloud to Snowflake
+- Build an example dbt Cloud project
+- Create a semantic model in dbt Cloud
+- Define metrics in dbt Cloud
+- Test and query metrics in dbt Cloud
+- Run a production job in dbt Cloud
+- Set up dbt Semantic Layer in dbt Cloud
+- Connect and query metrics with Google Sheets
+
+## Create new Snowflake worksheet and set up environment
+
+1. Log in to your [trial Snowflake account](https://signup.snowflake.com).
+2. In the Snowflake user interface (UI), click **+ Worksheet** in the upper right corner.
+3. Select **SQL Worksheet** to create a new worksheet.
+
+
+### Set up Snowflake environment
+
+The data used here is stored as CSV files in a public S3 bucket and the following steps will guide you through how to prepare your Snowflake account for that data and upload it.
+
+Create a new virtual warehouse, two new databases (one for raw data, the other for future dbt development), and two new schemas (one for `jaffle_shop` data, the other for stripe data).
+
+1. Run the following SQL commands one by one by typing them into the Editor of your new Snowflake SQL worksheet to set up your environment.
+
+2. Click **Run** in the upper right corner of the UI for each one:
+
+```sql
+-- Create a virtual warehouse named 'transforming'
+create warehouse transforming;
+
+-- Create two databases: one for raw data and another for analytics
+create database raw;
+create database analytics;
+
+-- Within the 'raw' database, create two schemas: 'jaffle_shop' and 'stripe'
+create schema raw.jaffle_shop;
+create schema raw.stripe;
+```
+
+## Load data into Snowflake
+Now that your environment is set up, you can start loading data into it. You will be working within the raw database, using the `jaffle_shop` and stripe schemas to organize your tables.
+
+1. Create customer table &mdash; First, delete all contents (empty) in the Editor of the Snowflake worksheet. Then, run this SQL command to create the customer table in the `jaffle_shop` schema:
+
+  ```sql
+  create table raw.jaffle_shop.customers
+  ( id integer,
+    first_name varchar,
+    last_name varchar
+  );
+  ```
+
+  You should see a ‘Table `CUSTOMERS` successfully created.’ message.
+
+2. Load data &mdash; After creating the table, delete all contents in the Editor. Run this command to load data from the S3 bucket into the customer table:
+
+  ```sql
+  copy into raw.jaffle_shop.customers (id, first_name, last_name)
+  from 's3://dbt-tutorial-public/jaffle_shop_customers.csv'
+  file_format = (
+      type = 'CSV'
+      field_delimiter = ','
+      skip_header = 1
+      );
+  ```
+
+  You should see a confirmation message after running the command.
```

Comment:
same as prev comment. 

consider removing lines like this. seems low value and superfluous (for conciseness and faster scanning)